bower.json

```diff
@@ -5,7 +5,7 @@
     "jquery": "1.11.2",
     "hogan": "3.0.2",
     "jquery-details": "https://github.com/mathiasbynens/jquery-details/archive/v0.1.0.tar.gz",
-    "digitalmarketplace-frontend-toolkit": "https://github.com/alphagov/digitalmarketplace-frontend-toolkit.git#v24.0.0",
+    "digitalmarketplace-frontend-toolkit": "https://github.com/alphagov/digitalmarketplace-frontend-toolkit.git#v23.3.0",
```

Comment:
Might be worth keeping the toolkit upgrade?

Reply:
I'd rather not so as this is an automatic revert of the entire PR. I'd rather not pick and choose which bits to keep if that's OK, as it makes it less clear what's happened.

Reply:
As long as we're 100% sure none of the more recent commits are relying on new bits of toolkit...